app/routes/companyInterest/components/Translations.ts

```diff
@@ -119,10 +119,10 @@ export const COLLABORATION_TYPES = {
     norwegian: 'Samarbeid med TIHLDE linjeforening',
     english: 'Event in collaboration with TIHLDE',
   },
-  collaboration_revue: {
-    norwegian: 'Samarbeid med Revyen',
-    english: 'Collaboration with the revue',
-  },
+  // collaboration_revue: {
+  //   norwegian: 'Samarbeid med Revyen',
+  //   english: 'Collaboration with the revue',
+  // },
```

Comment:
Perhaps remove these comments :grin:

Reply:
Nope. These are for eventually adding "Revy" as a collaborator later on